test/controllers/patients_controller_test.rb

```diff
@@ -12,6 +15,65 @@ class PatientsControllerTest < ActionDispatch::IntegrationTest
     @clinic = create :clinic
   end
 
+  describe 'index method' do
+    before do
+      # Not sure if there is a better way to do this, but just calling
+      # `sign_in` a second time doesn't work as expected
+      delete destroy_user_session_path
```

Comment:
haha yeah, that's something I should have sorted out when we moved onto rails 5 and the controller test class changed. Sorry to make you pay for my sins here, I'll file a backlog ticket to fix this up.